src/api/config/application.rb

```diff
@@ -10,8 +10,23 @@
 require 'sprockets/railtie'
 require 'rails/test_unit/railtie'
 
-# Assets should be precompiled for production (so we don't need the gems loaded then)
-Bundler.require(*Rails.groups(assets: ['development', 'test']))
+gemfile_in = File.expand_path('../Gemfile.in', __dir__)
+if File.exist?(gemfile_in)
+  # This gem disables Bundler's enforcement of exact gem versions
+  # in favour of basic constraint checks written in "Gemfile.in" file.
+  # It's not included in "Gemfile", so it should be installed explicitly
+  # by your system's package manager
+  # (as a dependency of OBS API package, for example)
+  # in order to allow system manager to take over responsibility of
+  # gem management from Bundler.
+  # Also, it's packager's responsibility to place appropriate
+  # "Gemfile.in" file to enable this feature.
```

Comment:
```suggestion
  # The bundler_ext rubygem disables bundler's enforcement of exact gem versions in `Gemfile.lock`
  # in favour of the basic constraints defined in the file `Gemfile.in`.
  # The bundler_ext rubygem is not part OBS, so you need to explicitly install it, however you
  # install gems on your system. Eg. with gem or your system package manager.
  # Then create the appropriate file with `cp Gemfile Gemfile.in`
  # WARNING: You will be on your own with problems if you do this, we only ensure our app works with the gems
  #                     specified in Gemfile.lock 
```

Reply:
Actually, "create the appropriate file with `cp Gemfile Gemfile.in`" is not exactly correct: `Gemfile.in` has to be patched after that. You can see how to do it in my `.spec` file.

Reply:
You mean the name changes? From activerecord to active_record?

Reply:
Yes. This is what I do in `.spec` file:
```
sed -i \
    -e "s/'activerecord'/'active_record'/" \
    -e "s/'actionmailer'/'action_mailer'/" \
    -e "s/'yajl-ruby'/'yajl'/" \
    -e "s/'sprite-factory'/'sprite_factory'/" \
    -e "s/'minitest-fail-fast'/'minitest-fail_fast'/" \
    %{buildroot}%{_datadir}/obs-api/Gemfile.in
```

Reply:
pushed in a slightly different way.